tasks/main.yml

```diff
@@ -59,7 +59,7 @@
     owner: root
     group: root
     mode: 0755
-  when: "{{ sysfs_uses_old_sysfsutils | bool == true }}"
+  when: sysfs_uses_old_sysfsutils
```

Comment:
Is it also possible to keep the `| bool`?

Reply:
To what extent? I can't see what it is intended to do?

Reply:
If you're expecting it to be put through a jinja2 filter and converted into a boolean, no but it is already boolean

Reply:
> If you're expecting it to be put through a jinja2 filter and converted into a boolean

Yes, that

You're right that in this role it's already a `bool`, but I remember that I had issues with something like this:

```yml
# supervisor role variables
---
supervisor_program_default_directory: /var/www/oefenweb.dev/app
supervisor_program_default_directory_log: "{{ supervisor_program_default_directory }}/log"
supervisor_program_default_directory_app: "{{ supervisor_program_default_directory }}/public_html/app"
supervisor_program_default_autostart: false
supervisor_program_default_autorestart: true
supervisor_program_default_stdout_logfile_maxbytes: 0
supervisor_program_default_stderr_logfile_maxbytes: 0
supervisor_program_default_user: www-data
supervisor_program_default_stopwaitsecs: 3600
supervisor_program_default_stopasgroup: true

supervisor_programs_present:
  cakephp-queue-foo:
    command: "{{ supervisor_program_default_directory_app }}/Console/cake queue.queue runworker"
    directory: "{{ supervisor_program_default_directory_app }}"
    environment: "APPLICATION_ENV={{ apache_mod_php_56_apache_default_environment_env }},APPLICATION_NAME=foo"
    autostart: "{{ supervisor_program_default_autostart }}"
    autorestart: "{{ supervisor_program_default_autorestart }}"
    stdout_logfile: "{{ supervisor_program_default_directory_log }}/queue.queue_foo.out"
    stdout_logfile_maxbytes: "{{ supervisor_program_default_stdout_logfile_maxbytes }}"
    stderr_logfile: "{{ supervisor_program_default_directory_log }}/queue.queue_foo.err"
    stderr_logfile_maxbytes: "{{ supervisor_program_default_stderr_logfile_maxbytes }}"
    user: "{{ supervisor_program_default_user }}"
    stopwaitsecs: "{{ supervisor_program_default_stopwaitsecs }}"
    stopasgroup: "{{ supervisor_program_default_stopasgroup }}"
```

In this case `stopasgroup` would be a string, instead of a bool

That's why i introduced the ` | bool == true`